### DISCUSSION TITLE
Height

### DISCUSSION THREAD

**Discussion by bsastregx** [nolimits4web/swiper#4237] [Q&A]
Hi!
I want to provide the slider a percentual height, for example in vw. Is this possible?
I have seen in the documentation that "height" is a number property (in px), but is not working.

`...    swiperOptions: {
        height: 100...`

That won´t work... 
Besides is not working, I need the height to be relative to the viewport widht. Is this possible?

**Comment by nolimits4web** [comment]
Yes, set in with CSS
